api-generation/src/main/scala/ai/h2o/sparkling/api/generation/common/IgnoredParameters.scala

```diff
@@ -23,15 +23,22 @@ object IgnoredParameters {
     "max_confusion_matrix_size", // Deep Learning
     "col_major", // Deep Learning
     "max_hit_ratio_k", // GBM, DRF, Deep Learning
-    "loading_name") // GLRM
+    "loading_name", // GLRM
+    "lambda_min_ratio") // GAM
 
   val implementedInParent: Seq[String] = Seq("training_frame", "validation_frame")
 
   val unimplemented = Seq(
     "__meta", // just for internal purposes
     "checkpoint", // GBM, DRF, XGBoost, Deep Learning
-    "interaction_pairs", // GLM
-    "pretrained_autoencoder") // DeepLearning
+    "interaction_pairs") // GLM, GAM
 
-  def all: Seq[String] = deprecated ++ implementedInParent ++ unimplemented
+  def common: Seq[String] = deprecated ++ implementedInParent ++ unimplemented
+
+  def all(algorithm: String): Seq[String] = algorithm match {
+    case "H2OKMeans" => common ++ Seq("response_column", "offset_column")
```

Comment:
Just note for the future: Param classes for all algos inherit for the base Model class which defines common parameters for all the params but there are a lot of cases where such param doesn't make much sense in the context of specific algo. For example KMeans and distribution etc. We could improve this in the future inside of this method.

WDYT?

Reply:
Do you propose to update `ModelParametersSchemaV3` and `Model.Parameters` class?

Reply:
Nope. What I meant is that we generate all parameters. For example, KmeansParametersV3 inherits from ParametersV3 which contains also parameters which do not make sense to KMeans. H2O has this issue, but we could actually fix it on our side by explicitly ignoring parameters for generation which do not make sense on specific algos

Reply:
Sorry, I bit slower today... So you propose to remove [common parameters](https://github.com/h2oai/sparkling-water/pull/2257/files/a904a14e3b6f3927778ed1e31d91850f5bb15a9a#diff-47e4179c219099038269faa68f455740R36) and list all relevant ignored parameters in  [case statement](https://github.com/h2oai/sparkling-water/pull/2257/files/a904a14e3b6f3927778ed1e31d91850f5bb15a9a#diff-47e4179c219099038269faa68f455740R39)?

Reply:
Nope, let's discuss on a call